controllers/validationController.js

```diff
@@ -41,14 +41,36 @@ exports.validateAdminPassword = (req, res, next) => {
     // console.log("adminPassword", adminPassword);
     const password = req.query.password
     // console.log("password", password);
-    if (password === adminPassword) {
-        // then good to go to players  
-        next();
-    } else {
+    
+    // get headers authorization
+    const authheader = req.headers.authorization;
+    console.log('req.headers.authorization', authheader);
+    // and distract only token password
+    const tokenPass = authheader.split(' ')[1]
+    console.log("tokenPass", tokenPass);
+
+    // get token password from frontend
+    // const authheaderPass = req.query.authorization
+    // console.log("authheaderPass", authheaderPass);
+    // password === adminPassword
+    if (tokenPass) {
```

Comment:
We just check whether the token exists or not but we don't check if this token is the correct password.